app/api/client.js

```diff
@@ -0,0 +1,7 @@
+import { create } from "apisauce";
+
+const apiClient = create({
+  baseURL: "https://ill-gold-parrot-sock.cyclic.app",
```

Comment:
I would prefer to place endpoints and other details in environment (.env) file or Config.

Reply:
I was thinking of refactoring this later, but why not now!?!.. updated :)